src/app/logout-page/logout-page.component.scss

```diff
@@ -0,0 +1 @@
+@import 'src/app/login-page/login-page.component.scss';
```

Comment:
This path changed.  On main it's `@import '../+login-page/login-page.component.scss';`   But this now starts with `src/app`